modules/javafx.graphics/src/main/java/com/sun/prism/es2/GLFactory.java

```diff
@@ -30,6 +30,7 @@
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 import java.util.HashMap;
```

Comment:
I think this import can be removed now.

Also check derivatives (aka. other `GLFactory`-ies extending this one)